packages/api/src/rich-text/unicode.ts

```diff
@@ -23,6 +25,10 @@ export class UnicodeString {
     return this.utf8.byteLength
   }
 
+  get graphemeLength() {
+    return graphemeLen(this.utf16)
+  }
```

Comment:
Definitely not necessary now, but it might be nice to eventually cache this result.